ddmrp/static/src/js/list_renderer_buffer_info.js

```diff
@@ -21,36 +22,30 @@ odoo.define("ddmrp.list_renderer_buffer_info", function(require) {
             this._super.apply(this, arguments);
         },
         _renderBodyCell: function(record, node, colIndex, options) {
-            var $td = this._super.apply(this, arguments);
-            options = node.attrs.options;
-            if (typeof options === "string") {
-                try {
-                    options = JSON.parse(options);
-                } catch (e) {
-                    console.log("Error while parsing JSON " + e);
-                }
+            var $td = this._super.apply(this, arguments),
+                node_options = node.attrs.options;
+            if (!_.isObject(options)) {
+                node_options = node_options ? pyUtils.py_eval(node_options) : {};
             }
-            if (options) {
-                if (options.color_from) {
-                    var color_field = options.color_from;
-                    this._updateNodeStyle($td, record, node, color_field);
+            if (node_options.color_from) {
```

Comment:
```suggestion
            if(node_options){
                  if (node_options.color_from) {
```
we need to check if node_option exists, otherwise it will broke other list views
Error:

```

Traceback:
Error: node_options is undefined
_renderBodyCell@http://localhost/ddmrp/static/src/js/list_renderer_buffer_info.js:31:17
OdooClass.extend/Class.include/</prototype[name]</<@http://localhost/web/static/src/js/core/class.js:123:38
_renderRow/$cells<@http://localhost/web/static/src/js/views/list/list_renderer.js:838:25
_renderRow@http://localhost/web/static/src/js/views/list/list_renderer.js:837:35
_renderRow@http://localhost/web/static/src/js/views/list/list_editable_renderer.js:1164:32
```

Reply:
Thanks. Fixed